wai-aria/role/invalid-roles.tentative.html

```diff
@@ -0,0 +1,28 @@
+<!doctype html>
+<html>
+<head>
+  <title>Tentative Invalid Role Verification Tests</title>
+  <script src="/resources/testharness.js"></script>
+  <script src="/resources/testharnessreport.js"></script>
+  <script src="/resources/testdriver.js"></script>
+  <script src="/resources/testdriver-vendor.js"></script>
+  <script src="/resources/testdriver-actions.js"></script>
+  <script src="/wai-aria/scripts/aria-utils.js"></script>
+  <meta charset="utf-8">
+</head>
+<body>
+
+<p>Tests invalid ARIA roles from <a href="https://w3c.github.io/aria/#document-handling_author-errors_roles">9.1 Roles - handling author errors</a>.</p>
+
+<!-- associationlist and related roles were briefly in the editors draft but removed pending: https://github.com/w3c/aria/issues/1662 -->
+<div role="associationlist" data-testname="associationlist is not a valid role" class="ex-generic">
+  <div role="associationlistkey" data-testname="associationlistkey is not a valid role" class="ex-generic">x</div>
+  <div role="associationlistvalue" data-testname="associationlistvalue is not a valid role" class="ex-generic">x</div>
+</div>
```

Comment:
Seems like these tentative tests aren't necessary, since no engine shipped them yet. I'll pull this new `tentative` file from the PR since the role invalidation turns out to be extraneous.

Reply:
Removed in https://github.com/web-platform-tests/wpt/pull/47608/commits/68bd87cbf488b78378cb16c330e8ab92145e017c